build/ks-controller-manager/Dockerfile

```diff
@@ -3,10 +3,10 @@
 # that can be found in the LICENSE file.
 FROM alpine:3.11
 
-COPY  /bin/cmd/controller-manager /usr/local/bin/
-
 RUN apk add --no-cache ca-certificates
 
+COPY  /bin/cmd/controller-manager /usr/local/bin/
```

Comment:
Are there any reasons why we should change it? It'd good if you can explain it. Thanks.